packages/file-search/src/browser/quick-editors-open.ts

```diff
@@ -0,0 +1,176 @@
+/********************************************************************************
+ * Copyright (C) 2019 Ericsson and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { inject, injectable } from 'inversify';
+import {
+    QuickOpenModel, QuickOpenItem, QuickOpenMode, PrefixQuickOpenService,
+    OpenerService, QuickOpenGroupItem, QuickOpenGroupItemOptions, QuickOpenItemOptions,
+    QuickOpenHandler, QuickOpenOptions
+} from '@theia/core/lib/browser';
+import { ApplicationShell } from '@theia/core/lib/browser/shell/application-shell';
+import { FileSystem } from '@theia/filesystem/lib/common/filesystem';
+import { WorkspaceService } from '@theia/workspace/lib/browser/workspace-service';
+import { MessageService } from '@theia/core/lib/common/message-service';
+import URI from '@theia/core/lib/common/uri';
+import { LabelProvider } from '@theia/core/lib/browser/label-provider';
+import { Command } from '@theia/core/lib/common';
+import * as fuzzy from 'fuzzy';
+import { CancellationTokenSource } from '@theia/core/lib/common';
+import { EditorWidget } from '@theia/editor/lib/browser';
+
+export const SHOW_ALL_OPENED_EDITORS: Command = {
+    id: 'workbench.action.showAllEditors',
+    category: 'View',
+    label: 'Show All Opened Editors'
+};
+
+@injectable()
+export class QuickEditorsOpenService implements QuickOpenModel, QuickOpenHandler {
```

Comment:
I'd keep it singular like the `QuickFileOpenService`, the file should be named `quick-editor-open`, and the class as `QuickEditorOpenService`.